<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TALoginWidget</class>
 <widget class="QWidget" name="TALoginWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TALoginWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="labelheadline">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_1">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>帐号：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="UID_lineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>密码：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="PWDlineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="memButton">
       <property name="text">
        <string>记住密码</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="moreButton">
         <property name="text">
          <string>设置</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox">
         <item>
          <property name="text">
           <string>在线</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>隐身</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>忙碌</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>离开</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>请勿打扰</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="loginButton">
         <property name="text">
          <string>登录</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <slots>
  <signal>setServer()</signal>
 </slots>
</ui>
